ci/before_install.sh

```diff
@@ -11,3 +11,7 @@ echo "inside $0"
 sudo apt-get update $APT_ARGS # run apt-get update for all versions
 
 true # never fail because bad things happened here
+
+if [ x"$DOC_BUILD" == x"" ]; then
+    exit 1
+fi
```

Comment:
yes, this was only for the testing from the temp commit (just so the others will all fail before installing pandas, so the doc build did not need to wait on all the others)